fixups/RegLegacyFixups/InitializeFixup.cpp

```diff
@@ -255,7 +349,35 @@ void InitializeConfiguration()
 
                             specItem.remediationRecords.push_back(recordItem);
                         }
+                        else if (type.compare(L"Redirect") == 0)
```

Comment:
Please use a case-insensitive compare.

Reply:
config.json is case sensitive, right?

Reply:
True.  Sorry, with string comparisons I always default to case-insensitive.

As long as PSF won't crash if a user puts in `redirection` with a little r I'm okay with this.

Reply:
Yah it won't crash but simply ignore